### PR TITLE
1: Contributing Guidelines

### PR DESCRIPTION
**Pull request by KaosElegent**
Finished the ticket.
The issue for this is #1 

image

### CODE REVIEW

**Comment by CrisH2307** [comment]
Good

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 ## PRs
 
+- Every PR should follow a naming convention. E.g. `#(IssueNumber): Issue Name`
```
Comment (KaosElegent) [inline CONTRIBUTING.md:12]: Oh that's a cool addition!

File: tsconfig.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
       }
     ],
     "paths": {
-      "@components/*": ["./app/components/*"]
+      "@components/*": ["./components/*"]
```
Comment (KaosElegent) [inline tsconfig.json:22]: Cool!
```suggestion

```

File: CONTRIBUTING.md (hunk 1 above)
Comment (KaosElegent) [reply CONTRIBUTING.md:12]: yup

**Review by Harshp258 — COMMENTED** [review]
works push it on main

**Review by CrisH2307 — APPROVED** [review]

**Review by raghavdotdev — APPROVED** [review]

**Review by aumsoni2002 — APPROVED** [review]

**Review by Harshp258 — APPROVED** [review]

**Pull request merged**